README.md

```diff
@@ -127,19 +127,18 @@ Splits should add up to equal the `debit_credit`.
 ```
  Account                       Type                
 ---------------------------------------
-checking                     asset               
-savings                      asset               
-credit_card                  liability           
+Checking                     asset               
```

Comment:
I'm curious as to why the asset accounts in these examples are capitalized but the other accounts are lower case. Is that just to differentiate the assets from other classes in the reporting output? We don't seem to enforce capitalization in the code.

Reply:
The actually account name is capitalized so that is why. You can see that in the rledger.yaml file.